examples/rllib/multiagent_exps/multiagent_figure_eight.py

```diff
@@ -42,13 +43,17 @@
         'noise': 0.2
     }),
     routing_controller=(ContinuousRouter, {}),
-    speed_mode='no_collide',
+    sumo_car_following_params=SumoCarFollowingParams(
```

Comment:
nit: if we're using double quotes elsewhere we should use them here

Reply:
If we want to transition to single quotes, I think we should fix lines as we come across them. Or alternatively we can make one PR where we modify all of them. I'm down for both